gp-templates/translation-set-form.php

```diff
@@ -15,7 +15,7 @@
 	<dt><label for="set[project_id]"><?php _e( 'Project', 'glotpress' ); ?></label></dt>
 	<dd><?php echo gp_projects_dropdown( 'set[project_id]', $set->project_id, null, null, true ); ?></dd>
 </dl>
-<?php echo gp_js_focus_on( 'set[locale]' ) . "\n"; ?>
+<?php echo gp_js_focus_on( 'set[locale]' ) . "\n"; // WPCS: sanitization ok. ?>
```

Comment:
That's the wrong comment.

 `gp_js_focus_on()` should to be added to https://github.com/GlotPress/GlotPress-WP/blob/develop/phpcs.ruleset.xml#L16. But first `$html_id` in https://github.com/GlotPress/GlotPress-WP/blob/develop/gp-includes/template.php#L235 needs to be escaped with `esc_js()`.